infrastructure/main.tf

```diff
@@ -43,11 +43,23 @@ data "vault_generic_secret" "db_password" {
 }
 
 data "vault_generic_secret" "staff_email" {
-  path = "secret/${var.vault_section}/cmc/claim-store/staff_email"
+  path = "secret/${var.vault_section}/cmc/claim-store/staff-email"
 }
 
-data "vault_generic_secret" "rpa_email" {
-  path = "secret/${var.vault_section}/cmc/claim-store/rpa_email"
+data "vault_generic_secret" "rpa_email_sealed_claim" {
```

Comment:
hashi vault is no longer in use,
all secrets have been migrated to azure key vault,
the locations for the migrated ones you need are:

```
rpa-email
rpa-email-sealed-claim
rpa-email-more-time-requested
rpa-email-response
rpa-email-ccj
```